packages/plugin-credentials/src/credentials/common.js

```diff
@@ -493,9 +514,8 @@ export default {
   _receiveSupertoken(supertoken) {
     const scopes = [
       `spark:kms`,
-      apiScope
+      filterScope(`spark:kms`, this.config.scope)
```

Comment:
so, line 19 was a mistake. by pulling it from the environment instead of config (which typically pulls it from the environment), i made it a lot hard to use multiple instances of the sdk. Feel free to fix that if you need to. 

apiScope was intended to be all the scopes except `spark:kms`.